src/harbour-tuschbox.cpp

```diff
@@ -53,12 +53,16 @@ int main(int argc, char *argv[])
 
     qmlRegisterType<QQuickFolderListModel>("harbour.tuschbox.Tuschbox", 1, 0, "FolderListModel");
 
+
     QQuickView *view = SailfishApp::createView(); // I get a white background with this.
     view->setSource(SailfishApp::pathTo("qml/harbour-tuschbox.qml"));  // So I do this ;)
 
     FM *fileAction = new FM();
     view->engine()->rootContext()->setContextProperty("_fm", fileAction);
 
+    Settings *settings = new Settings();
```

Comment:
QSettings is nice. However I use already a database for settings(QtQuick.LocalStorage). I would like to use that instead of QSettings here.
See https://github.com/llelectronics/tuschbox/blob/master/qml/pages/js/db.js